bin/start.sh

```diff
@@ -34,12 +24,96 @@ then
     fi
   fi
 
-  cd ${COMPONENT_HOME}/share/zlux-app-server/bin
+  # used for relativeTo plugins
+  export ZLUX_ROOT_DIR=$COMPONENT_HOME/share
+else
+ # dev env
+ . ./validate.sh
+ COMPONENT_HOME=$(cd ../..; pwd)
+
+  # used for relativeTo plugins
+  export ZLUX_ROOT_DIR=$COMPONENT_HOME
 fi
 
+ZLUX_APP_SERVER_DIR=${ZLUX_ROOT_DIR}/zlux-app-server
+cd ${ZLUX_APP_SERVER_DIR}/bin
+
+. ./init/node-init.sh
 . ./utils/setup-logs.sh
 
-# Done to prevent app-server from making a logfile since we will be making one here instead.
-export ZLUX_NO_LOGFILE=1
+# Get config path or fail
+if [ -z "${ZWE_CLI_PARAMETER_CONFIG}" ]; then
+  echo "ZWE_CLI_PARAMETER_CONFIG is not defined. Only defaults will be used."
+  echo "To customize, rerun script with it defined to a list of paths to zowe.yaml files such as FILE(/yaml1.yaml):FILE(/path/to/yaml2.yaml)"
+  if [ -e "${HOME}/.zowe/zowe.yaml" ]; then
+    echo "Found and using ${HOME}/.zowe/zowe.yaml"
+  else
+    mkdir -p ${HOME}/.zowe
+    cp ${ZLUX_APP_SERVER_DIR}/defaults/serverConfig/defaults.yaml ${HOME}/.zowe/zowe.yaml
+  fi
+  CONFIG_FILE="FILE(${HOME}/.zowe/zowe.yaml):FILE(${ZLUX_APP_SERVER_DIR}/defaults/serverConfig/defaults.yaml)"
+else
+  CONFIG_FILE="FILE(${ZWE_CLI_PARAMETER_CONFIG}):FILE(${ZLUX_APP_SERVER_DIR}/defaults/serverConfig/defaults.yaml)"
```

Comment:
code comment: this will be zowe-merged.yaml in production, therefore no FILE() on the input, safe to wrap.